SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -4035,6 +4086,39 @@ class CSolver {
                                     CSolver** solver_container,
                                     CConfig* config) { }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
+   */
+   inline virtual void SetEddyViscFirstPoint(CGeometry *geometry,
+                              CSolver** solver_container,
+                              CConfig* config) { }
+
+   /*!
+    * \brief A virtual member.
+    * \param[in] geometry - Geometrical definition of the problem.
+    * \param[in] solver_container - Container vector with all the solutions.
+    * \param[in] config - Definition of the particular problem.
+    */
+   inline virtual void Setmut_LES(CGeometry *geometry,
```

Comment:
Similarly for these methods, if they are part of the preprocessing of the NS solver, they should be private methods of that solver.
The abstraction (i.e. the virtual function) we want in CSolver is "Preprocessing", everything that is physics-specific should concern only the derived classes (Euler, NS, etc.) and fall in the abstract categories of Preprocessing, postprocessing, iteration, etc. which are the only hooks we should use to augment the functionality of the solvers.